ng-ncc-app/src/app/common/services/notes.service.ts

```diff
@@ -309,7 +309,9 @@ export class NotesService {
             }, details),
 
             // Action Diary note...
-            this.recordActionDiaryNote(noteMessage, call_nature)
+            callType !== null ? (callTypes.includes(callType) ?
```

Comment:
Perhaps create a private method for this, as very similar code is used above.
Looks good otherwise.

Reply:
Okay. Makes sense. 

Thanks.